src/common/utils/time-monitor/time-monitor.ts

```diff
@@ -25,15 +25,13 @@ import { maxTimeQueryForCube } from "../query/max-time-query";
 
 export class TimeMonitor {
```

Comment:
What's interesting, we have one TimeMonitor for whole turnilo instance. That is one of few places where we use source name as a unique key across possible multiple clusters. Such situation would break turnilo instance. Something to consider in the future.